tests/javascript/Tiltfile

```diff
@@ -0,0 +1,52 @@
+# Some common test configurations to easily get up and running with Tilt for Tests
+# (see docs: https://docs.tilt.dev/tests_in_tilt.html)
+
+
+def _test_js(name, pkmanager, dir, deps=None, with_install=False, project_root='.', extra_args=None):
+    install_deps = deps_for_package_manager(pkmanager, project_root)
+    resource_deps = []
+    if with_install:
+        install_resource = pkmanager + '-install'
+        install_cmd = 'cd {dir} && {pkmanager} install'.format(
+            dir=dir, pkmanager=pkmanager)
+        local_resource(install_resource, install_cmd, install_deps, allow_parallel=True)
+        resource_deps = [install_resource]
+
+        # double check that we could find package.json etc. correctly
+        for f in install_deps:
+            if not os.path.exists(f):
+                warn('''expected JS config file "{}" not found -- did you pass project_root correctly when setting up JS tests?
```

Comment:
except it's not really called a "config" file, is it? ugh what's the word?

Reply:
This has a nice list of the various files that can be used for this: https://docs.npmjs.com/cli/v7/configuring-npm

But I don't see any sign of a name!

Would it be weird to just say "expected file {} not found"?